Message TTL (#4785)
EnMasseProject/enmasse: documentation/modules/ref-ttl.adoc

The `ttl` field is used to restrict the effective http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-properties[`absolute-expiry-time`] of

> any message put to a queue or topic. The `maximum` and `minimum` values, defined in milliseconds will adjust the TTL value of an incoming messages to that address if

Reviewer: ```suggestion
any message put to a queue or topic. The `maximum` and `minimum` values are defined in milliseconds. The system adjusts the TTL value of an incoming message to a particular address based on these values:
```

> the following ways:

Reviewer: ```suggestion

```

> * If a messages arrives at the address with a TTL value that is greater than the `maximum`, its TTL is changed to the maximum value.

Reviewer: ```suggestion
* If a messages arrives at the address with a TTL value that is greater than the `maximum` value, the system changes the message TTL to the maximum value.
```

> * If a messages arrives with a TTL value that is less than the `minimum`, the TTL is changed to the minimum.

Reviewer: ```suggestion
* If a message arrives at the address with a TTL value that is less than the `minimum` value, the system changes the message TTL to the minimum value.
```

> Messages arriving without a TTL defined are considered to have a TTL value of infinity when considered by these rules.

Reviewer: ```suggestion
Messages that arrive without a TTL defined are considered to have a TTL value of infinity.
```

> Expired messages will be automatically removed from the `queue` or `subscription` periodically. This occurs every 30

Reviewer: ```suggestion
Every 30 seconds, expired messages are automatically removed from the `queue` or `subscription`.
```

> seconds.

Reviewer: ```suggestion

```
